src/server/pps/cmds/cmds.go

```diff
@@ -164,7 +164,8 @@ If the job fails, the output commit will not be populated with data.`,
 
 	var pipelineName string
 	var inputCommitStrs []string
-	var history string
+	var listPipelineHistory string
+	var listJobHistory string
```

Comment:
This change is fine but shouldn't change behavior. Only one of `list pipeline` or `list job` will run per invocation of `pachctl` so the variable can't persist between invocations.

Reply:
Yep. The issue here is... trying to set different defaults for "history" in `list pipeline` and `list job` does not work if we have the same variable. The second invocation of `Flags().StringVar(&history...` sets the default value. Which in this case was from `list pipeline`